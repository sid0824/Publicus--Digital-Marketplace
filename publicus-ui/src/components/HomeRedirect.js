import React from 'react';
import {Redirect} from "react-router-dom";
import RoutePaths from "../core/RoutePaths";

// The component used to redirect to the home page when a url was given that does not match
// any existing page
export default () => (
  <Redirect
    to={RoutePaths.Home}
  />
)