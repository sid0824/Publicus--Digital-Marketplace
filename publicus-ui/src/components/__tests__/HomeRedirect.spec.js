import toJson from 'enzyme-to-json';
import React from "react";
import {shallow, configure} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import HomeRedirect from "../HomeRedirect";

configure({ adapter: new Adapter() });

describe('Home Redirect', () => {

  beforeEach(() => {

  });

  afterAll(() => {

  });

  test('should render home redirect', () => {
    const wrapper = shallow(
      <HomeRedirect />
    );
    expect(toJson(wrapper)).toMatchSnapshot();
  });

});