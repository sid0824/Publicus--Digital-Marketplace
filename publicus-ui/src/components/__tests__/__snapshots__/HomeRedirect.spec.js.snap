// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Redirect should render home redirect 1`] = `
<Redirect
  to="/"
/>
`;
